### ISSUE TITLE
the anatomical Variable in the Slide Level Feature Extraction Code

### ISSUE THREAD

**Issue by Zhcyoung** [hms-dbmi/CHIEF#51]
```python
full_path = r'./Downstream/Tumor_origin/src/feature/tcga/TCGA-LN-A8I1-01Z-00-DX1.F2C4FBC3-1FFA-45E9-9483-C3F1B2B7EF2D.pt' 
features = torch.load(full_path, map_location=torch.device('cpu'))
anatomical = 13
with torch.no_grad():
    x, tmp_z = features, anatomical
    result = model(x, torch.tensor([tmp_z]))
    wsi_feature_emb = result['WSI_feature']  ###[1,768]
    print(wsi_feature_emb.size())
```

It is unclear what the variable `anatomical` represents. The value `13` is assigned to it, but there is no explanation about its meaning. 

**Questions:**
1. Is `anatomical` a specific value that needs to be set based on certain criteria, or can it be arbitrarily chosen?
2. If it is not arbitrary, how should I determine the correct value for `anatomical`?

Could you please provide clarification on what `anatomical` represents and how to determine the correct value for it in this context? 

Thank you!

**Comment by Zhcyoung** [comment]
Oh, I found the meaning of anatomical!

**Comment by SeaTalk** [comment]
what does it mean?😳😳

**Comment by Zhcyoung** [comment]
> an?😳😳

The anatomical variable and its code can be found here:  CHIEF/configs/anatomic_mapping.yaml
`Anatomic:
    'brain':0,
    'breast':1,
    'bladder':2,
    'kidney':3,
    'prostate':4,
    'testis':5,
    'lung':6,
    'pancreas':7,
    'liver':8,
    'skin':9,
    'ovary':10,
    'cervix':11,
    'uterus':12,
    'colon':13,
    'esophagus':14,
    'stomach':15,
    'thyroid':16,
    'adrenal gland':17,
    'soft tissue':18`

**Comment by Zhcyoung** [comment]
I think it is derived from the patch features: 
`image = Image.open("./exsample/exsample.tif")
image = trnsfrms_val(image).unsqueeze(dim=0)
with torch.no_grad():
    patch_feature_emb = model(image) # Extracted features (torch.Tensor) with shape [1,768]
    print(patch_feature_emb.size())`
